src/components/ControlInput.vue

```diff
@@ -147,6 +151,9 @@ export default {
         shouldShowSendButton() {
             return this.$state.ui.is_touch || this.$state.setting('showSendButton');
         },
+        showEmojiPicker() {
```

Comment:
This should be renamed to `shouldShowEmojiPicker` for consistency

Reply:
fixed with force push ;)